src/OSPSuite.Presentation/DTO/Charts/CurveDTO.cs

```diff
@@ -127,31 +130,31 @@ public static IEnumerable<IBusinessRule> All
             get
             {
                yield return dataDimensionMatchesXAxis;
-//               yield return dataDimensionMatchesYAxis;
+               yield return dataDimensionMatchesYAxis;
                yield return dataDimensionMatchesYAxisType;
-//               yield return canConvertYDataToYAxisUnits;
                yield return canConvertYAxisTypeToYAxisUnits;
             }
          }
 
          private static IBusinessRule dataDimensionMatchesXAxis { get; } = CreateRule.For<CurveDTO>()
             .Property(x => x.xData)
-            .WithRule((curveDTO, column) => curveDTO.dataDimensionMatchesXAxis())
+            .WithRule((curveDTO, column) => curveDTO.dataDimensionMatchesXAxis(column))
```

Comment:
I need to use the column being passed to ensure that the validation is done as expected